csaf_2.1/prose/README.md

```diff
@@ -0,0 +1,47 @@
+# The CSAF 2.1 Prose Folder
+
+This place offers access to the editable sources of the v2.1 CSAF specification (to be).
+
+In the `share` folder there are the user facing delivery items that offer layout and navigation
+optimized for online viewing per 
+
+- a typical web interface of a version control server (like Codeberg, GitHub, GitLab, or SourceHut) - the Markdown file
+- any typical browser (like Brave, Chrome, Edge, Firefox, or Safari) - the HTML file
+
+Inside the `edit` folder we build these delivery items from the source files (also in Markdown format, but
+split by concerns, verifiable per syntax, and offering clean structural constructs for definition lists etc.
+instead of the specific idioms mixed in for ease of use in specific reading tools).
+
+The `csaf-v2.1-editor-draft.md` in this folder is an intermediate file and may vanish at any point in time.
+
+In general, we try to keep this file identical to the `share/csaf-v2.1-draft.md` file.
+
+The latter is generated from the source files below `edit/src/` as collected per `edit/etc/bind.txt` through
+the `edit/bin/volatile.py` script (for now).
+
+The way to the HTML is a three-step process:
+
+First calling pandoc (inside the `edit` folder) as:
+
+```console
+% pandoc -f gfm+definition_lists -t html build/tmp.md --columns=345  --css style/base.css --css style/skin.css  \
+  --standalone -o build/tmp.html \
+  --metadata title="Common Security Advisory Framework Version 2.0"
```

Comment:
Well, it should be when we all agree, that his starting point represents 2.0 so that we are all supportive of any difference as an intended 2.1 "feature" and not some oversight or random glitch.